vmdb/app/models/ems_event/parsers/kubernetes.rb

```diff
@@ -0,0 +1,21 @@
+module EmsEvent::Parsers::Kubernetes
+  def self.event_to_hash(event, ems_id = nil)
+    log_header = "MIQ(#{name}.event_to_hash) ems_id: [#{ems_id}]"
+
+    $log.debug("#{log_header} event: [#{event.inspect}]")
+
+    event_type = "#{event[:kind].upcase}_#{event[:reason].upcase}"
```

Comment:
Even though now for kubernetes they are constructed automatically it's probably something I won't try to maintain in the future. For example I think that it would be best to generate `NODE_READY` events regardless of the container solution (kubernetes / amazon).

In other words I wouldn't try to have provider-specific events (well unless that's what you have today and you're satisfied with it).